wordpress-mobile/WordPress-Android
# Hiding lazy loading preferences for unsupported sites.

Fixes #11021 

This PR hides Lazy Load Images option from Site Settings on unsupported sites, such as:
- wpcom sites
- self-hosted/atomic sites without a connected jetpack
- self-hosted/atomic sites with a jetpack connected to a different account than currently logged in one (those sites are treated as self-hosted without jetpack)

To test:
- Select normal wpcom site and check Site Settings, make sure Lazy Load Images is not there.
- Select a self-hosted/atomic site without connected jetpack and check Site Settings, make sure Lazy Load Images is not there.
- Select a self-hosted/atomic site WITH jetpack connected to a currently logged in wpcom account, check Site Settings and make sure Lazy Load Images IS there and works (it sometimes takes a bit of time to propagate settings to the endpoint and sync them with the client).


PR submission checklist:
